iothub/service/src/Exceptions/IotHubStatusCode.cs

```diff
@@ -204,5 +202,13 @@ public enum ErrorCode
         /// IoT hub is currently unable to process the request. This is a transient, retryable error.
         /// </summary>
         ServiceUnavailable = 503001,
+
+        /// <summary>
+        /// The operation failed because the IoT hub has been suspended. 
+        /// </summary>
+        /// <remarks>
+        /// This is likely due to exceeding Azure spending limits. To resolve the error, check the Azure bill and ensure there are enough credits.
+        /// </remarks>
+        HubSuspended, // to-do: brycewang - check the error code returned by the hub service while sending requests to a suspended hub.
```

Comment:
to-do: I am checking this with the service team for a 6-digit code returned back in the response content

Reply:
Added "IotHubSuspended = 400020" enum to IotHubErrorCode. Complete!